todo.py

```diff
@@ -56,8 +91,10 @@ def main():
             new_todo = input("Enter a todo: ")
             add_todo(todo_list, new_todo)
         elif choice == 3:            
-            index_to_delete = get_choice("Enter the index to complete: ")
-            delete_todo(todo_list, index_to_delete)
+            item_to_delete = input("Enter the item to complete: ")
```

Comment:
This could get tricky for a user - if they've typed in long-ish todos, looking it up by the todo description itself would be tedious.